src/environments/environment.ts

```diff
@@ -4,8 +4,8 @@
 
 export const environment = {
   production: false,
-  BASE_URL: "https://stage.skintechno.com/api/",
-  RESOURCE_URL: "https://stage.skintechno.com/",
+  // BASE_URL: "https://stage.skintechno.com/api/",
+  // RESOURCE_URL: "https://stage.skintechno.com/",
```

Comment:
@themuhammadattaurrehman 
revert you changes from this, or don't commit while committing other files